api/src/services/__tests__/meetingsService.ts

```diff
@@ -162,7 +167,106 @@ describe('meetingsService', () => {
         )
       ).toEqual({ id: meetingNoteId });
     });
-
+    it('should resolve to null if no meeting note id exists', async () => {
```

Comment:
I don't understand what situation this description is referring to. It looks like it's the situation where the insert statement doesn't insert a new row. Is there a better way to communicate that to the reader?

Reply:
Yes, there's a better way to communicate that to the reader. When the meeting note responds with an empty array, it should resolve to null.